guides/queries/tracing.md

```diff
@@ -113,19 +113,19 @@ end
 
 ## Skylight
 
-To add [Skylight](https://www.skylight.io) instrumentation:
+To add [Skylight](https://www.skylight.io) instrumentation, you may either enable the [GraphQL probe](https://www.skylight.io/support/getting-more-from-skylight#graphql) or use [ActiveSupportNotificationsTracing](/queries/tracing.html#activesupportnotifications).
 
 ```ruby
-class MySchema < GraphQL::Schema
-  use(GraphQL::Tracing::SkylightTracing)
-end
+# config/application.rb
+config.skylight.probes << "graphql"
 ```
 
-
 <div class="monitoring-img-group">
   {{ "/queries/skylight_example.png" | link_to_img:"skylight monitoring" }}
```

Comment:
Should we update this example screenshot of monitoring, too, to reflect the new GraphQL probe, or is it basically the same?

Reply:
We just [replaced the image](https://github.com/rmosolgo/graphql-ruby/blob/06d7c8bec07a7bdd8aaf62b52d1a0368187cb116/guides/queries/skylight_example.png) (`skylight_example.png`) with an updated screenshot of what the GraphQL probe looks like in action in the Skylight UI.

Let me know if you think that's ok!